### DISCUSSION TITLE
Pattern fill for WebGLVectorLayerRenderer

### DISCUSSION THREAD

**Discussion by tommy1x** [openlayers/openlayers#15298] [Q&A] [answered]
I'm using WebGLVectorLayerRenderer to render some topojson files.
Is it possible to fill using a pattern (e.g. a hatch pattern) instead of a colour?

**Comment by jahow** [comment, marked as answer]
This is now possible with #15279, please let us know here how things work out for you or if you encountered any issue, thanks!

**Reply by tommy1x** [reply]
Thanks, have got it working now with some help from #15299. I understand how it works now but the styling part and adding the webglrenderer was difficult at first without proper documentation.

Also, I'm getting a console error in this dev version with my webgl vector layer which I wasn't getting in 8.1.0:
Internal error: 0x00000502: Vertex buffer is not big enough for the draw call.

This also seems to sometimes cause a layer's pattern/colour to flash or fail to render (but still shows the stroke).

**Reply by jahow** [reply]
Could you please share a minimum reproduction of your issue, so that it can be investigated? thanks!

**Reply by tommy1x** [reply]
Is there a sandbox or something I can easily fork to add this? I tried some but had some problems.

**Comment by ahocevar** [comment]
@tommy1x You can `npm install ol@dev`.
